edx-frontend-build-tools/config/webpack.prod.config.js

```diff
@@ -3,25 +3,25 @@
 const Merge = require('webpack-merge');
 const commonConfig = require('./webpack.common.config.js');
 const path = require('path');
-const glob = require('glob');
+const Dotenv = require('dotenv-webpack');
 const webpack = require('webpack');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 const HtmlWebpackNewRelicPlugin = require('html-webpack-new-relic-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const PurgecssPlugin = require('purgecss-webpack-plugin');
 const NewRelicSourceMapPlugin = require('new-relic-source-map-webpack-plugin');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin; // eslint-disable-line prefer-destructuring
 const PostCssRtlPlugin = require('postcss-rtl');
 const PostCssAutoprefixerPlugin = require('autoprefixer');
 const CssNano = require('cssnano');
+const cwd = process.cwd();
```

Comment:
Warning: this may be different than __dirname, right?  Does it pre-suppose that someone's running the command from a particular directory?

Reply:
Like __dirname is guaranteed not to change, regardless of where the program was run from... I wonder if cwd() is the same

Reply:
process.cwd() will be the root project path. I've updated the code a lot but that's the idea.